dandiapi/api/management/commands/import_dandisets.py

```diff
@@ -0,0 +1,82 @@
+from urllib.parse import urljoin
+
+from django.db import transaction
+import djclick as click
+import requests
+
+from dandiapi.api.models import Dandiset, Version
+
+
+@transaction.atomic
+def import_versions_from_response(api_url: str, version_api_response: dict, dandiset: Dandiset):
+    """Import versions given a response from /api/dandisets/{identifier}/versions/."""
+    for result in version_api_response['results']:
+        # get the metadata of this version
+        metadata = requests.get(
+            urljoin(
+                api_url,
+                f'/api/dandisets/{result["dandiset"]["identifier"]}/versions/{result["version"]}/',
+            )
+        ).json()
+
+        click.echo(f'  Importing version "{result["version"]}"')
+
+        version = Version(
+            dandiset=dandiset,
+            name=result['name'],
+            version=result['version'],
+            doi=result.get('doi'),
+            status=result['status'],
+            metadata=metadata,
+        )
+        version.save()
+
+    # Handle API pagination
+    if version_api_response.get('next'):
+        next_versions = requests.get(version_api_response.get('next')).json()
+        import_versions_from_response(api_url, next_versions, dandiset)
+
+
+@transaction.atomic
+def import_dandisets_from_response(api_url: str, dandiset_api_response: dict):
+    """Import dandisets given a response from /api/dandisets/."""
+    for result in dandiset_api_response['results']:
+
+        # Check if dandiset with this name already exists. If it does, skip it.
+        # I can't think of a better way to uniquely identify dandisets
+        # without being able to use identifiers.
+        existing_dandiset = Version.objects.filter(name=result['draft_version']['name'])
+        if existing_dandiset.first():
```

Comment:
```suggestion
        if existing_dandiset.exists():
```
`exists` is generally preferred to `first`, since postgres can run optimize it a bit more. However, in this case it probably doesn't make a difference, or might even slow it down, since using `first` will cache the value for later. No change necessary, just FYI

Reply:
I would say favor clarity over possible optimization (that is, use `exists()` here).